app/controllers/sessions_controller.rb

```diff
@@ -26,7 +31,8 @@ def create
     else
       sign_in user
       user.update_attribute(:lastlogin, Time.now)
-      redirect_to root_path
+      redirect_back_or(root_path)
+      #redirect_to root_path
```

Comment:
any reason to keep this commented out vs. deleting it?

Reply:
no reason, other than preservation of the endangered cruft species :) 

I'll delete it